packages/core/services/oidc-provider/oidc-provider.js

```diff
@@ -0,0 +1,133 @@
+const Redis = require('ioredis');
+
+const { Errors: E } = require('moleculer-web');
+const RedisAdapter = require('./adapter');
+const baseConfig = require('./base-config');
+const fetch = require('node-fetch');
+
+module.exports = {
+  name: 'oidc-provider',
+  settings: {
+    baseUrl: undefined,
+    frontendUrl: undefined,
+    redisUrl: undefined,
+    cookieSecret: 'cookie-secret'
```

Comment:
I suppose, this value needs to be overriden, maybe make it caps to amplify that?

Reply:
I added caps, and put this secret in an env variable